build/common/installer/scripts/tomlparser-geneva-config.rb

```diff
@@ -225,6 +225,14 @@ def get_command_windows(env_variable_name, env_variable_value)
   return "[System.Environment]::SetEnvironmentVariable(\"#{env_variable_name}\", \"#{env_variable_value}\", \"Process\")" + "\n" + "[System.Environment]::SetEnvironmentVariable(\"#{env_variable_name}\", \"#{env_variable_value}\", \"Machine\")" + "\n"
 end
 
+def is_configure_geneva_env_vars()
+  is_configure = false
+  if (@geneva_logs_integration && !@multi_tenancy) || (@geneva_logs_integration && @multi_tenancy && !@infra_namespaces.empty?)
+    is_configure = true
+  end
+  return is_configure
+end
```

Comment:
can line 228 to 234 be changed to return @geneva_logs_integration && (!@multi_tenancy || !@infra_namespaces.empty?)   since they both checking @geneva_logs_integration